app/templates/ember-community-survey-2020.hbs

```diff
@@ -5,114 +5,375 @@
   @surveyLogoSrc="/images/survey/logo2020.svg"
 >
   <p>
-    A project to learn who is using Ember and how.
+    We gathered responses from over 1,000 developers for the 2020 survey, despite coinciding with the global pandemic breakout and subsequent quarantine for many countries across the globe. This year we also included outreach to developers who don't use Ember yet, which made up for approximately 10% of the total responses we received.
   </p>
-
-  <div class="mt-3">
-    <a
-      class="es-button"
-      href="https://tilde.wufoo.com/forms/2020-emberjs-community-survey/"
-      rel="noopener noreferrer"
-      target="_blank"
-    >
-      Take the survey now
-    </a>
-  </div>
 </EmberCommunitySurvey::Introduction>
 
 <EmberCommunitySurvey::Section
   @hasDarkBackground={{false}}
-  @sectionId="about-the-survey"
-  @sectionTitle="About the Survey"
+  @sectionId="recommending-ember"
+  @sectionTitle="Recommending Ember"
 >
+  <:charts>
+    <Highcharts
+      @chartOptions={{this.SS_Q101.options}}
+      @data={{this.SS_Q101.data}}
+    />
+  </:charts>
+
   <:body>
-    <div class="layout">
-      <div class="lg:col-2">
-        <img
-          alt="Tomster and Friends"
-          class="survey-image"
-          src="/images/survey/photo1.jpg"
-        >
-
-        <h3>
-          Totally Anonymous
-        </h3>
-
-        <p>
-          No personal identification is attached to survey responses. Last year, over 1300 participants shared information about themselves, their businesses, and applications.
-        </p>
-      </div>
+    <p>
+      We asked respondents how likely they were to recommend Ember to others, on a scale of 1-5. The mean recommendation score is 4; approximately 80% of people who use Ember are highly likely to recommend it to others.
+    </p>
+    <p>
+      Participants reported their reasoning for their score as free-form text, and below, we've provided a sample of responses alongside their corresponding recommendation score.
+    </p>
+    <p>
+      As in previous years, we did not observe a great deal of variety in the written responses that corresponded to values of three or less. Similarly, we saw the opposite effect in the upper ranges—that is, <strong>community members simply have a lot of different reasons to recommend Ember</strong>!
```

Comment:
I think using similarly can be confusing here. Initially, I thought it was referring to "not a great deal of variety" instead of "as in previous years." How about

```suggestion
      As in previous years, we did not observe a great deal of variety in the written responses that corresponded to values of three or less. Like previous years, we saw the opposite effect in the upper ranges—that is, <strong>community members simply have a lot of different reasons to recommend Ember</strong>!
```

?